frontends/concrete-python/concrete/fhe/mlir/processors/assign_bit_widths.py

```diff
@@ -13,266 +13,346 @@
 
 class AssignBitWidths(GraphProcessor):
     """
-    Assign a precision to all nodes inputs/output.
-
-    The precisions are compatible graph constraints and MLIR.
-    There are two modes:
-        - single precision: where all encrypted values have the same precision.
-        - multi precision: where encrypted values can have different precisions.
+    AssignBitWidths graph processor, to assign proper bit-widths to be compatible with FHE.
```

Comment:
Not sure why the doc was removed. It's still valid.

Reply:
I've pretty much started from scratch, that's the reason. Let me introduce it back.